torch/ao/sparsity/parametrization.py

```diff
@@ -0,0 +1,10 @@
+from torch import nn
+
+class MulBy(nn.Module):
+    def __init__(self, other):
+        super().__init__()
+        self.other = other
+
+    def forward(self, x):
+        assert self.other.shape == x.shape
+        return self.other * x
```

Comment:
Suggest having basic tests here:
1. Create sparseLinearModule, ensure that weights, bias are instantiated correctly [from_dense method works]
2. Ensure that from_float works correctly and the quantized sparse module has the correct weights
3. Ensure that sparseLinearModule can be serialized (both state_dict and jit serialization)
4. Ensure that sparseLinearModule is symbolically traceable.